### DISCUSSION TITLE
What can I do about mem swap size

### DISCUSSION THREAD

**Discussion by Dentonknifeworks** [aaronwmorris/indi-allsky#683] [Q&A] [answered]
![memswp](https://user-images.githubusercontent.com/64991600/230625238-826182f8-a36e-4830-87ed-ccafa3a7c4f6.png)
Can I adjust the swap size?

**Comment by aaronwmorris** [comment, marked as answer]
You should decrease your kernel swappiness to 1 like in https://github.com/aaronwmorris/indi-allsky/wiki/Memory-Optimizations

```
echo "vm.swappiness = 1" | sudo tee /etc/sysctl.d/90-swappiness.conf

sudo sysctl --system
```

You can clear swap an raspi (without rebooting) by doing the following:
```
sudo dphys-swapfile swapoff

sudo dphys-swapfile swapon
```

Setting swappiness to 1 just sets a preference to decreasing file cache to free memory instead of swapping pages to disk.
